x-pack/test/observability_functional/apps/observability/alerts/bulk_actions.ts

```diff
@@ -1,204 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-import expect from '@kbn/expect';
-import { FtrProviderContext } from '../../../ftr_provider_context';
-
-async function asyncForEach<T>(array: T[], callback: (item: T, index: number) => void) {
-  for (let index = 0; index < array.length; index++) {
-    await callback(array[index], index);
-  }
-}
-
-export default ({ getService, getPageObjects }: FtrProviderContext) => {
-  const esArchiver = getService('esArchiver');
-  const observability = getService('observability');
-
-  const retry = getService('retry');
-
-  // Hide the WorkFlow filter, but keep its code as required in https://github.com/elastic/kibana/issues/117686
-  describe.skip('Observability alerts / Bulk actions', function () {
-    this.tags('includeFirefox');
-    before(async () => {
-      await esArchiver.load('x-pack/test/functional/es_archives/observability/alerts');
-      await esArchiver.load('x-pack/test/functional/es_archives/infra/metrics_and_logs');
-      await esArchiver.load(
-        'x-pack/test/apm_api_integration/common/fixtures/es_archiver/apm_8.0.0'
-      );
-    });
-
-    after(async () => {
-      await esArchiver.unload('x-pack/test/functional/es_archives/infra/metrics_and_logs');
-      await esArchiver.unload('x-pack/test/functional/es_archives/observability/alerts');
-      await esArchiver.unload(
-        'x-pack/test/apm_api_integration/common/fixtures/es_archiver/apm_8.0.0'
-      );
-    });
-
-    describe('When user has all priviledges for logs app', () => {
-      before(async () => {
-        await observability.users.setTestUserRole(
-          observability.users.defineBasicObservabilityRole({
-            logs: ['all'],
-          })
-        );
-        await observability.alerts.common.navigateToTimeWithData();
-      });
-
-      after(async () => {
-        await observability.users.restoreDefaultTestUserRole();
-      });
-
-      it('logs checkboxes are enabled', async () => {
```

Comment:
@fkanout I would still keep this test, but change the expectation, for example checkbox is not visible